apps/meteor/packages/rocketchat-i18n/i18n/pl.i18n.json

```diff
@@ -292,12 +292,6 @@
   "add-livechat-department-agents_description": "Uprawnienie do dodawania agentów omnichannel do działów",
   "add-oauth-service": "Dodaj usługę Oauth",
   "add-oauth-service_description": "Uprawnienie do dodawania nowej usługi Oauth",
-  "add-team-channel": "Dodaj zespół Channel",
```

Comment:
Is this necessary to be removed? 🤔

Reply:
afaik lingohub doesn't update existing translations, so I decided to delete the ones we have so that they are regenerated.
Anyways, I'm checking if it'd be ok to create a new permission and totally erase this one.

Reply:
I decided to replace the `add-team-channel` permission by a new one `move-room-to-team`. The current permission's name can be misleading.